app/assets/javascripts/discourse/app/widgets/user-menu.js

```diff
@@ -149,11 +156,35 @@ createWidget("user-menu-links", {
 
     glyphs.push(this.profileGlyph());
 
+    schedule("afterRender", () => {
+      const tabList = document.querySelector(".glyphs");
+      const mousetrap = new Mousetrap(tabList);
+      const maxTabNumber = glyphs.length - 1;
+
+      mousetrap.bind(["right", "left"], (e) => {
```

Comment:
This seems a bit risky to me. Every time the widget is rendered you bind this? It's never unbound either? 

It might be better to bind this in the component that mounts the widget and send it via app event.